### DISCUSSION TITLE
Welcome to osm-data-model Discussions!

### DISCUSSION THREAD

**Discussion by joto** [osmlab/osm-data-model#9] [Announcements]
##  Welcome to OSM data model discussions!

This is the place to discuss ideas and plans around evolving the OSM data model. There are three categories available for discussion things related to the study, but you can always also discuss anything and everything in the "General" category.

**Comment by testman42** [comment]
Context for this discussion: https://blog.openstreetmap.org/2023/01/04/reminder-call-for-feedback-on-the-data-model/

Otherwise, the new proposal for the data model looks good to me.
